main_routers/vrm_router.py

```
@@ -0,0 +1,382 @@
+            "success": True,
+            "animations": animations
+        })
+    except Exception as e:
+        import traceback
+        error_detail = traceback.format_exc()
+        logger.exception("获取VRM动画列表失败")
+        logger.error(f"错误详情: {error_detail}")
+        # 返回更详细的错误信息以便调试
+        error_message = str(e)
+        error_type = type(e).__name__
+        # 提取关键错误信息（最后几行堆栈跟踪）
```

Ruff (0.14.11)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)